client/state/selectors/test/is-email-blacklisted.js

```diff
@@ -0,0 +1,40 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { isEmailBlacklisted } from '../';
+
+const email = 'foo@bar.baz';
+
+const state = {
+	siteSettings: {
+		items: {
+			123: {
+				blacklist_keys: 'mail@mail.com\ntest@example.com\nfoo@bar.baz\nyadda@yadda.yadda',
+			},
+			456: {
+				blacklist_keys: 'mail@mail.com\ntest@example.com\nyadda@yadda.yadda',
+			},
+			789: {
+				blacklist_keys: '',
+			},
+		},
+	},
+};
+
+describe( 'isEmailBlacklisted()', () => {
```

Comment:
you may want to add tests for `null` or absent `email` and missing site settings.